ush/setup.sh

```diff
@@ -1654,6 +1654,26 @@ fi
 #
 #-----------------------------------------------------------------------
 #
+# Make sure that DO_SAVE_INPUT is set to a valid value.
+#
+#-----------------------------------------------------------------------
+#
+check_var_valid_value "DO_SAVE_INPUT" "valid_vals_DO_SAVE_INPUT"
+#
+# Set DO_SAVE_INPUT to either "TRUE" or "FALSE" so we don't have to consider
+# other valid values later on.
+#
+DO_SAVE_INPUT=${DO_SAVE_INPUT^^}
+if [ "$DO_SAVE_INPUT" = "TRUE" ] || \
+   [ "$DO_SAVE_INPUT" = "YES" ]; then
+  DO_SAVE_INPUT="TRUE"
+elif [ "$DO_SAVE_INPUT" = "FALSE" ] || \
+     [ "$DO_SAVE_INPUT" = "NO" ]; then
+  DO_SAVE_INPUT="FALSE"
+fi
+#
+#-----------------------------------------------------------------------
+#
```

Comment:
Please check 471-485 to see if you can add this one as to the list that makes the same changes.